app/jobs/scheduled/autoexpire_user_api_keys.rb

```diff
@@ -0,0 +1,15 @@
+module Jobs
+
+  class AutoexpireUserApiKeys < Jobs::Scheduled
```

Comment:
naming wise I would recommend AutoExpireUserApiKeys here, and the setting should be expire_user_api_keys_days